### PR TITLE
Allow redshift in SQL resource

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

File: into/tests/test_pydot.py [hunk 1]
```diff
@@ -6,4 +7,9 @@ def test_create_pydot_image():
         dot_graph()
     except:
         pass
-
+    finally:
+        for ext in ('pdf', 'png'):
+            try:
+                os.remove('conversions.%s' % ext)
+            except:
+                pass
```
Comment (mrocklin) [inline into/tests/test_pydot.py]: FWIW I actually intended this side effect.  At the moment often use the graph for debugging.  I admit that this is a bit dirty/invasive though.

Comment (cpcloud) [inline into/tests/test_pydot.py]: how about I add a little shell script to do it?

Comment (mrocklin) [inline into/tests/test_pydot.py]: The `into/dot.py` file is itself a script.  We can remove the auto-generation of graphs.  I can put this into my personal workflow and stop polluting everyone elses.

**Comment by cpcloud** [comment]
ok 2 merge then?

**Pull request merged**